docs/download-smartos.md

```diff
@@ -1,31 +1,34 @@
 # Download SmartOS
 
+## Latest Release
+
 * [Download SmartOS ISO][download-iso] - For running in Virtual Box or VMware
```

Comment:
Can also be used on Real HW if it has a CD-ROM drive (Triton's ISO requires DVD-ROM, alas.  :) )